airflow/cli/commands/task_command.py

```diff
@@ -560,6 +561,9 @@ def task_test(args, dag=None):
     if args.task_params:
         passed_in_params = json.loads(args.task_params)
         task.params.update(passed_in_params)
+        if isinstance(task, DecoratedOperator):
+            task.op_args = []
+            task.op_kwargs.update(passed_in_params)
```

Comment:
I'm not 100% sure that this is the best place to override this. Probably the args should rather be overridden somewhere closer to `execute` (e.g., https://github.com/apache/airflow/blob/main/airflow/operators/python.py#L174).

I lack clear understanding what `params` are / should be used for as opposed to `op_args` and `op_kwargs`, so overriding here looked like a safer change to me.